aries/agents/rust/mediator/src/aries_agent/client/transports.rs

```diff
@@ -30,7 +28,6 @@ pub trait AriesTransport {
 }
 
 pub struct AriesReqwest {
-    pub response_queue: VecDeque<Value>,
     pub client: reqwest::Client,
 }
```

Comment:
Seems like the Reqwest wrapper doesn't need to exist anymore. But that seems out of scope, feel free to address it separately.

Reply:
Done